source/downloads.rst

```diff
@@ -2,12 +2,36 @@
 Download
 ========
 
+.. Note::
 
-`Sponge alpha development builds are now available! <https://repo.spongepowered.org/maven/org/spongepowered/sponge/>`__
+    **Changelogs** for Sponge coremod, SpongeVanilla and SpongeAPI are now available on the respective Downloads pages, along with links for Javadocs and sources.
 
 
-If you'd like, you may compile SpongeVanilla from its GitHub repository; however, keep in mind that it may not be usable
-at this point.
+Sponge (coremod)
+~~~~~~~~~~~~~~~~
+
+* **Status:** Alpha. The first alpha build of Sponge coremod (#428, Inspired Wallaroo) was released on April 25th, 2015.
+
+*Sponge coremod is soon to be known as SpongeForge.*
+
+`Sponge coremod alpha development builds are now available! <https://repo.spongepowered.org/maven/org/spongepowered/sponge/>`__
+
+
+SpongeVanilla
+~~~~~~~~~~~~~
+
+* **Status:** Alpha. The first alpha build of SpongeVanilla (DEV#17) was released on September 13th, 2015.
+
+`SpongeVanilla alpha development builds are now available! <https://repo.spongepowered.org/maven/org/spongepowered/spongevanilla/>`__
+
+For the more adventurous, they may also be compiled from their respective GitHub repositories:
 
 * Sponge (coremod): `Sponge on GitHub <https://github.com/SpongePowered/Sponge>`_
 * SpongeVanilla: `SpongeVanilla on GitHub <https://github.com/SpongePowered/SpongeVanilla>`_
```

Comment:
I suggest moving the "For the adventurous" comment to the top of the page and including the respective github link within the artifact's section so that the sponge coremod link is under the sponge coremod heading.